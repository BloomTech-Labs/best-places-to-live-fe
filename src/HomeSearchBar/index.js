import React from 'react';
import { Link } from 'react-router-dom'
import { makeStyles } from '@material-ui/core/styles';
import AppBar from '@material-ui/core/AppBar';
import Toolbar from '@material-ui/core/Toolbar';
// import Typography from '@material-ui/core/Typography';  * not being used.. not sure if we need it later so just commenting it out
import Button from '@material-ui/core/Button';
import IconButton from '@material-ui/core/IconButton';
import MenuIcon from '@material-ui/icons/Menu';
import TextField from '@material-ui/core/TextField';

const useStyles = makeStyles(theme => ({
    textField: {
        display: 'flex',
        maxWidth: 500,
        margin: '0 auto',
        marginTop: 100,
    },

}));

export default function HomeSearchBar() {
    const classes = useStyles();

    return (
        <div className={classes.homeSearch}>
            <TextField
                id="outlined-search"
                label="Enter an Address, City, or Zip Code"
                type="Search "
                className={classes.textField}
                margin="normal"
                variant="outlined"
            />
        </div>
    );
        
    
}