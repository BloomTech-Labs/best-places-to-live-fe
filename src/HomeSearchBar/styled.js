import styled from "styled-components";



export const Container = styled.div`
background-image:url('https://images.unsplash.com/photo-1449034446853-66c86144b0ad?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1350&q=80');
background-size: cover;
background-repeat: no-repeat;
background-position: center; 
position: relative;
height: 60vh;
width: 100%;
display:flex;
flex-flow: row wrap
justify-content: center;
`;

export const SearchContainer = styled.span`
display:flex;
flex-direction:column;
align-items:center;
width: auto;
height: 20%;
padding-top: 11%;
    h1 {
    color: white;
    }
    
    input{
    width: 100%;
    padding: 12px;
    border: none;
    border-radius: 4px;
    margin: 5px 0;
    opacity: 0.85;
    display: inline-block;
    font-size: 17px;
    line-height: 20px;
    outline:none;
    text-decoration: none; 
    }
`;






//   @media @tablet {
//       height: 40vh;
//   }

//   span {
//       padding-top: 11%;
//       .flex-box(column, wrap);
//       .custom-layout(center, center);

//       button {
//           .join-button(120px, 46px);
//       }
//   }

//   #banner-txt{
//       font-size: @giant-text;
//       font-weight: 400;
//       color: white;

//       @media @tablet {
//           font-size: @medium-text;
//           font-weight: 400;
//       }
//   }

//   #banner-sub-text{
//       color: whitesmoke;

//       @media @tablet {
//           font-size: @medium-text;
//       }
//   }
// }