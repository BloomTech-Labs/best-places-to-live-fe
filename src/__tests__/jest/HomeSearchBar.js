import React from "react";
import HomeSearchBar from "../../HomeSearchBar";
import renderer from "react-test-renderer";

describe("<HomeSearchBar />", () => {
  it("Matches Snapshot", () => {
    const tree = renderer.create(<HomeSearchBar />).toJSON();
    // @ts-ignore
    expect(tree).toMatchSnapshot();
  });
});
