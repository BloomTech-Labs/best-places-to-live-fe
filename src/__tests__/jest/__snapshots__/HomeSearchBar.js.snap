// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeSearchBar /> Matches Snapshot 1`] = `
<div
  className="sc-bdVaJa etUyXk"
>
  <span
    className="sc-bwzfXH eTcapO"
  >
    <h1>
       Discover a place you'll love to live
    </h1>
    <form
      onSubmit={[Function]}
    >
      <input
        id="searchTerm"
        label="Enter City or State"
        onChange={[Function]}
        placeholder="Enter City or State"
        type="search"
        value=""
      />
    </form>
  </span>
</div>
`;
